src/components/header/gh-version.js

```diff
@@ -1,36 +1,12 @@
 import { h } from 'preact';
-import { useEffect, useState } from 'preact/hooks';
-import { fetchRelease } from '../../lib/github';
-import config from '../../config.json';
 
-const URL = 'https://github.com/' + config.repo;
-
-let VERSION = '10.0.0';
-if (PRERENDER) {
-	VERSION = require('../../../package.json').dependencies.preact.replace(
-		'^',
-		''
-	);
-}
+const VERSION = '10.0.4';
+const URL = `https://github.com/preactjs/preact/releases/tag/${VERSION}`;
```

Comment:
Could we fetch all the tags and sort them based on semver rather than latest?

Reply:
That would work too. I have a branch ready for that, but I'm running into rate limiting issues with the API. Looks like there is a bit more work involved here for a proper longterm solution.

Reply:
Nevermind, I added it.